CHANGES_NEXT_RELEASE

```diff
@@ -15,6 +15,9 @@
 - Fix: bug when tenants are over 44 bytes long (#2811)
 - Add: Support for null values in string filters (#2359)
 - Fix: Partial sanity check for custon url, only if the url contains no replacements using ${xxx} (#2280, #2279)
+- Fix: if entity::type is given but as an empty string, an error is returned (#1985)
```

Comment:
I'd suggest to explain in which operations this fix has been done. Maybe this?

"Fix: if subject.entities*.type is given but as an empty string in POST /v2/subscriptions or PATCH /v2/subscriptions/{sub} , an error is returned (#1985)"